### ISSUE TITLE
[v2.17.0] Missing client TS types Aggregate*Args

### ISSUE THREAD

**Issue by steschwa** [prisma/prisma#5749]
Since upgrading to v2.17.0 from v2.15.0 for both `@prisma/client` and `prisma` i get an TypeScript error stating that the `Aggregate*Args` types are missing:

<img width="2032" alt="VSCode screenshot - missing TS types" src="https://user-images.githubusercontent.com/24265130/108521601-33082a00-72cc-11eb-944a-e30404940f44.png">

After some digging around in `index.d.ts` I've found the type `AggregateUser` which in my understanding should replace  `AggregateUserArgs` in  `aggregateUser<T extends AggregateUserArgs>`.  

Unfortunately downgrading to **v2.15.0** has a similar error.

I've used:
- `prisma migrate dev --preview-feature` (added Table `Query`) after removing the `prisma/migrations` folder (as suggested [here](https://github.com/prisma/prisma/issues/5367), although i did not switch DB providers)
- `prisma generate`
- `prisma db seed --preview-feature`

My **prisma.schema**:

```prisma
datasource db {
    provider = "mysql"
    url      = env("DATABASE_URL")
}

generator client {
    provider = "prisma-client-js"
}

model User {
    id        Int       @id @default(autoincrement())
    name      String?   @unique
    email     String    @unique
    password  String
    createdAt DateTime  @default(now())
    updatedAt DateTime? @updatedAt
}

enum MediaSource {
    LOCAL
    EXTERNAL
}

model Media {
    id           Int         @id @default(autoincrement())
    path         String
    source       MediaSource @default(LOCAL)
    mime         String
    previewImage String?     @db.Text
    createdAt    DateTime    @default(now())
    updatedAt    DateTime?   @updatedAt
    product      Product?
    market       Market?
}

model Product {
    id         Int        @id @default(autoincrement())
    name       String     @unique
    media      Media?     @relation(fields: [mediaId], references: [id])
    mediaId    Int?
    createdAt  DateTime   @default(now())
    updatedAt  DateTime?  @updatedAt
    expenses   Expense[]
    categories Category[]
}

model Market {
    id        Int       @id @default(autoincrement())
    name      String    @unique
    media     Media?    @relation(fields: [mediaId], references: [id])
    mediaId   Int?
    createdAt DateTime  @default(now())
    updatedAt DateTime? @updatedAt
    expenses  Expense[]
}

model Expense {
    id        Int      @id @default(autoincrement())
    product   Product  @relation(fields: [productId], references: [id])
    productId Int
    market    Market?  @relation(fields: [marketId], references: [id])
    marketId  Int?
    cost      Float
    amount    Int      @default(1)
    date      DateTime @db.Date
    createdAt DateTime @default(now())
}

model Category {
    id        Int       @id @default(autoincrement())
    name      String    @unique
    color     String?
    createdAt DateTime  @default(now())
    updatedAt DateTime? @updatedAt
    products  Product[]
}

model Query {
    id         Int       @id @default(autoincrement())
    color      String?
    collection String
    query      String    @db.Text
    createdAt  DateTime  @default(now())
    updatedAt  DateTime? @updatedAt
}
```

**Comment by steschwa** [comment]
Alright i think i found the problem.

It's seems that there's a problem with database tables named `Query`. I was receiving the same error as in https://github.com/prisma/prisma-client-js/issues/577 after declaring the missing TS types as dummies (`type AggregateUserArgs = {}`).

I first thought that it's a problem with my GraphQL query, which it wasn't. I'm not sure how this all relates to not generating the `Aggregate*` types (the `include` property for eager loading was also missing), but it seems to be the source of the problem.

I renamed my table name in the schema and everything works fine:
```prisma
// Previous
// model Query {
model CalculatedQuery {
    id         Int       @id @default(autoincrement())
    color      String?
    collection String
    query      String    @db.Text
    createdAt  DateTime  @default(now())
    updatedAt  DateTime? @updatedAt
}
```

**Comment by wtachau** [comment]
Same issue— would be nice to have some docs on not naming a table `Query`

**Comment by SevInf** [comment]
This issue is fixed and will be available in Prisma 5.1
